recipe/meta.yaml

```diff
@@ -60,7 +60,7 @@ requirements:
     - {{ cdt('libice-devel') }}          # [linux]
     - {{ cdt('libuuid-devel') }}         # [linux]
   host:
-    - python=3.7.8
+    - python=3.7
```

Comment:
Python's version will be picked from `conda_build_config.yaml`.
```suggestion
    - python
```

Reply:
OK, I can fix that